ui/packages/atlasmap/src/Views/ColumnMapperView/Actions/ImportAction.tsx

```diff
@@ -13,20 +13,72 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */
-import { Button, Tooltip } from '@patternfly/react-core';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  Tooltip,
+} from '@patternfly/react-core';
 import React, { FunctionComponent, useEffect, useRef } from 'react';
-
+import { DocumentType } from '@atlasmap/core';
 import { ImportIcon } from '@patternfly/react-icons';
 import { useFilePicker } from 'react-sage';
+import { useToggle } from '../../../impl/utils';
 
 export interface IImportActionProps {
   id: string;
-  onImport: (selectedFile: File) => void;
+  onImport: (selectedFile: File, docType: DocumentType) => void;
 }
-export const ImportAction: FunctionComponent<IImportActionProps> = ({
-  id,
-  onImport,
-}) => {
+
+type importDocumentType = {
+  documentType: DocumentType;
+  typeName: string;
```

Comment:
Can you add `inspectionType` here? in that way `DocumentType` would be able to stay like before, isolated from schema/instance. Otherwise we need to remove `inspectionType` completely from both of UI and runtime to avoid further confusion.

Reply:
Considering that the Syndesis is using atlasmap-maven-plugin to perform offline inspection, removing `InspectionType` from runtime is more risky. So we'd be better to keep it.